pkg/collector/helm_collector.go

```diff
@@ -0,0 +1,75 @@
+package collector
+
+import (
+	"github.com/Azure/aks-periscope/pkg/interfaces"
+	"github.com/Azure/aks-periscope/pkg/utils"
+	"path/filepath"
+)
+
+// HelmCollector defines a Helm Collector struct
+type HelmCollector struct {
+	BaseCollector
+}
+
+var _ interfaces.Collector = &HelmCollector{}
```

Comment:
What is this throw-away variable for?

Reply:
It's to verify that the HelmCollector class implements the Collector interface. This assignment tests if a type implements an interface. It's going to give you compile errors if you don't implement the Collector interface.